client/src/components/LandingPage/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import './index.css'
+
+const LandingPage = () => {
+  return (
+    <>
+      <div class='vertical'>
```

Comment:
```suggestion
      <div className='vertical'>
```
use `className` instead of `class`